imgui.cpp

```diff
@@ -4483,6 +4483,13 @@ void ImGui::EndFrame()
         g.CurrentWindow->Active = false;
     End();
 
+    for (auto window : g.Windows)
+    {
+        // Draw empty node background (currently can only be the Central Node)
+        if (window->DockNode && window->DockNode->IsEmpty() && window->DockNode->IsVisible && !(window->DockNode->GetMergedFlags() & ImGuiDockNodeFlags_PassthruCentralNode))
+            window->DrawList->AddRectFilled(window->DockNode->Pos, window->DockNode->Pos + window->DockNode->Size, GetColorU32(ImGuiCol_DockingEmptyBg));
+    }
+
```

Comment:
Why iterating on windows and not visible nodes?

Reply:
Oh, I couldn't find the list of nodes before, I'll fix that up and also make it a proper array not using auto.